packages/circuit-ui/components/Card/components/Header/Header.tsx

```diff
@@ -48,7 +48,8 @@ export type CardHeaderProps = {
    * Additional data that is dispatched with the tracking event.
    */
   tracking?: TrackingProps;
-} & CloseProps;
+} & CloseProps &
+  HTMLAttributes<HTMLDivElement>;
```

Comment:
Same here.

Reply:
Unfortunately the `CardHeader` isn't a styled component, it handles logic around the close button. I think it makes sense to extend the HTML element in this case to make the types more accurate, WDYT?